### PR TITLE
Ms 2 ms of pgn with holes with empty ouetr boundary efif

### PR DESCRIPTION
**Pull request by efifogel**
## Summary of Changes

The pull request does the following:
1. Adds supports for C++11 to the test suite of the 2D Minkowski sum (Minkowski_sum_2) package.
2. Enhances the test 'test_minkowski_sum_with_holes' so that it can be fed with different combinations of methods to compute the Minkowski sum of polygons with holes and test cases. This is necessary to be able to test degenerate cases using certain methods but not other methods (cause, e.g., the other methods do not support these degenerate cases).
3. Fixes the code that constructs the 2D Minkowski sum of polygons with holes, where the outer boundary of one or both of the summands are empty, using the reduced convolution method.

## Release Management

* Affected package(s): Minkowski_sum_2
* Issue(s) solved (if any): A segmentation occurs when computing the Minkowski sum of polygons with holes that have empty outer boundaries using the reduced convolution method.
* Feature/Small Feature (if any):

### CODE REVIEW

**Comment by sloriot** [comment]
One warning: `Minkowski_sum_2/test/Minkowski_sum_2/test_minkowski_sum_with_holes.cpp:156:12: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]`

**Comment by efifogel** [comment]
Thanks
Fixed and pushed.

   ____  _        ____             _
  /_____/_) o    /__________  __  //
 (____ (   (    (    (_/ (_/-(-'_(/
                         _/

**Comment by sloriot** [comment]
Test all red.

**Comment by efifogel** [comment]
grrr, forgot to commit the new test case...
Fixed and pushed.

   ____  _        ____             _
  /_____/_) o    /__________  __  //
 (____ (   (    (    (_/ (_/-(-'_(/
                         _/

**Comment by sloriot** [comment]
Successfully tested in CGAL-4.13-Ic-3

File: Minkowski_sum_2/test/Minkowski_sum_2/CMakeLists.txt [hunk 1]
```diff
@@ -6,6 +6,16 @@ project( Minkowski_sum_2_Tests )
...
+list(FIND CMAKE_CXX_COMPILE_FEATURES cxx_generalized_initializers has_cpp11)
+if (has_cpp11 LESS 0)
+  message(STATUS "NOTICE: These examples requires a C++11 compiler and will not be compiled.")
+  return()
+endif()
+
+# Use C++11 for this directory and its sub-directories.
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED TRUE)
+
```
Comment (lrineau) [inline Minkowski_sum_2/test/Minkowski_sum_2/CMakeLists.txt]: Efi, would you mind fixing that? It was already merged in `master`, but I think you could add that "fix" in the PR about Arr_2: https://github.com/CGAL/cgal/pull/3093

**Comment by efifogel** [comment]
All done (as you suggested---it's in the mine/Aos_2-ric_remove_edges-efif
branch).

   ____  _        ____             _
  /_____/_) o    /__________  __  //
 (____ (   (    (    (_/ (_/-(-'_(/
                         _/

**Pull request merged**